README.md

```diff
@@ -13,12 +13,32 @@ Create and manage Azure Virtual Machines directly from VS Code.
 
 ### Create New Virtual Machine
 
-![CreateVirtualMachine](resources/CreateVirtualMachine.gif)
+- Create an Azure VM with the default image of "Ubuntu 18.04-LTS" and size "Standard D2s V3"
+
+![CreateVirtualMachine](resources/CreateAzureVM.png)
+
+![CreateVirtualMachine2](resources/CreateAzureVM2.png)
+
+- Automatically generates and configures an SSH key used to remotely connect to new Azure VM

```

Comment:
I'd scrap the bullet point stuff. There's not that many bullets and full sentences would make more sense IMO.